### PR TITLE
Reorganize `astro workspace` docs by breaking out subcommands into separate docs

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/1971

Still adding some "Related documentation" links and redirects, but wanted to get the base content out there so that we can agree on the structure.

### CODE REVIEW

File: astro/cli/astro-workspace-list.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+sidebar_label: "astro workspace list"
+title: "astro workspace list"
+id: astro-workspace-list
+description: Reference documentation for astro workspace list.
+hide_table_of_contents: true
+---
+
+Generates a list of all Workspaces that you have access to.
```
Comment (paolaperaza) [inline astro/cli/astro-workspace-list.md:9]:
```suggestion
Generates a list of all Workspaces within your current Organization that you have access to.
```

Yes? Might be a better way of saying this

File: astro/cli/astro-workspace-list.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+## Usage
+
+Run `astro workspace list` to see the name and Workspace ID for each Workspace to which you have access.
```
Comment (paolaperaza) [inline astro/cli/astro-workspace-list.md:13]:
```suggestion
Run `astro workspace list` to see the name and Workspace ID for each Workspace to which you have access. You can use this information to run other CLI commands against your desired Workspace.
```

Something like that? Just to make it clear that's the real use case

File: astro/cli/astro-workspace-switch.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+description: Reference documentation for astro workspace switch.
+hide_table_of_contents: true
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+:::info  
+
+The behavior and format of this command differs depending on what Astronomer product you're using. Use the following tabs to change product contexts.
```
Comment (paolaperaza) [inline astro/cli/astro-workspace-switch.md:14]:
```suggestion
The behavior and format of this command differs depending on which Astronomer product you're using. Use the following tabs to change product contexts. 
```

File: astro/cli/astro-workspace-switch.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+The behavior and format of this command differs depending on what Astronomer product you're using. Use the following tabs to change product contexts. 
+
+:::
+
+Switch between Workspaces.
+
+## Usage 
+
+Run `astro workspace switch <workspace-id>` to switch between Workspaces. On Astro, you can run `astro workspace` and the CLI prompts you to select from a list of Workspaces to which you have access.
```
Comment (paolaperaza) [inline astro/cli/astro-workspace-switch.md:22]: It says there are tabs here -- are there? I suspect this one should given the command is shared across both (but it maybe isn't actually different at all?)

File: astro/cli/astro-workspace-user-add.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+description: Reference documentation for astro workspace.
+hide_table_of_contents: true
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+:::info  
+
+The behavior and format of this command differs depending on what Astronomer product you're using. Use the following tabs to change product contexts.
```
Comment (paolaperaza) [inline astro/cli/astro-workspace-user-add.md:14]:
```suggestion
The behavior and format of this command differs depending on which Astronomer product you're using. Use the following tabs to change product contexts. 
```

File: astro/cli/astro-workspace-user-add.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+
+<Tabs
+    defaultValue="astro"
+    values={[
+        {label: 'Astro', value: 'astro'},
+        {label: 'Software', value: 'software'},
+    ]}>
+<TabItem value="astro">
+
+Invite an existing user to your current Astronomer Workspace.
```
Comment (paolaperaza) [inline astro/cli/astro-workspace-user-add.md:26]:
```suggestion
Add an existing Organization user to your current Astronomer Workspace as an Admin, Editor, or Viewer. You must be a Workspace Admin to perform this action. After you add the user, their information appears in the Access tab as a new entry in the Members list. To access the Workspace, the user needs to accept the invitation sent by email and log in.

This command is functionally equivalent to using the **Add member** button in the Workspace Settings view of the Cloud UI.
```

Maybe too much? Feel free to cut me here

File: astro/cli/astro-workspace-user-add.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+
+```sh
+astro workspace user add <email> 
+```
+
+## Options
+
+| Option    | Description                                          | Valid Values                                                                               |
+| --------- | ---------------------------------------------------- | ------------------------------------------------------------------------------------------ |
+| `<email>` | The email for the user you want to invite or update. | Any valid email                                                                            |
```
Comment (paolaperaza) [inline astro/cli/astro-workspace-user-add.md:38]:
```suggestion
| `<email-address>` | The email address of the user that you want to add to the Workspace. | Any valid email address                                                                            |
```

File: astro/cli/astro-workspace-user-add.md [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+| `--email` (_Required_) | The user's email                                                                                                                                      | Any valid email address                                                                                                 |
+| `--workspace-id`       | The Workspace that the user is added to. Specify this flag if you want to create a user in a Workspace that is different than your current Workspace. | Any valid Workspace ID                                                                                                  |
+| `--role`               | The role assigned to the user.                                                                                                                        | Possible values are `WORKSPACE_VIEWER`, `WORKSPACE_EDITOR`, and `WORKSPACE_ADMIN`. Default value is `WORKSPACE_VIEWER`. |
+
+## Related documentation
+
+- [Manage Workspaces and Deployments on Astronomer](https://docs.astronomer.io/software/manage-workspaces)
+- [Manage User Permissions on Astronomer](https://docs.astronomer.io/software/workspace-permissions)
```
Comment (paolaperaza) [inline astro/cli/astro-workspace-user-add.md:68]: Add https://docs.astronomer.io/astro/add-user?

File: astro/cli/astro-workspace-user-list.md [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+
+<Tabs
+    defaultValue="astro"
+    values={[
+        {label: 'Astro', value: 'astro'},
+        {label: 'Software', value: 'software'},
+    ]}>
+<TabItem value="astro">
+
+Outputs a list of all users with access to your current Workspace.
```
Comment (paolaperaza) [inline astro/cli/astro-workspace-user-list.md:26]: Should we specify what information is included? e.g. email address, role, etc.

File: astro/cli/astro-workspace-user-list.md [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+
+```sh
+astro workspace user list
+```
+
+## Options
+
+| Option           | Description                                                                                                                                             | Possible Values        |
+| ---------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------- |
+| `--workspace-id` | The Workspace that you're searching in. Specify this flag if you want to search for users in a Workspace that is different than your current Workspace. | Any valid Workspace ID |
```
Comment (paolaperaza) [inline astro/cli/astro-workspace-user-list.md:55]:
```suggestion
| `--workspace-id` | The Workspace for which you want to list users. Specify this flag if you want to search for users in a Workspace that is different than your current Workspace. | Any valid Workspace ID |
```

File: astro/cli/astro-workspace-user-list.md [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+| `--email`        | The email for the user you're searching for.                                                                                                            | Any string             |
```
Comment (paolaperaza) [inline astro/cli/astro-workspace-user-list.md:56]:
```suggestion
| `--email`        | The email address for the user you're searching for.                                                                                                            | Any string             |
```

Comment (paolaperaza) [reply astro/cli/astro-workspace-user-list.md:56]: Is either email or user required? Probs not I suppose? Maybe specify what is required?

File: astro/cli/astro-workspace-user-remove.md [hunk 12]
```diff
@@ -0,0 +1,64 @@
...
+
+<Tabs
+    defaultValue="astro"
+    values={[
+        {label: 'Astro', value: 'astro'},
+        {label: 'Software', value: 'software'},
+    ]}>
+<TabItem value="astro">
+
+Remove a user from your current Workspace.
```
Comment (paolaperaza) [inline astro/cli/astro-workspace-user-remove.md:26]: We actually barely have any info here: https://docs.astronomer.io/astro/add-user#update-or-remove-a-workspace-user

Anything else we should specify about this process? Does the user immediately lose access to the Deployment and the Airflow UI? But they're still part of the Organization?

File: astro/cli/astro-workspace-user-remove.md [hunk 13]
```diff
@@ -0,0 +1,64 @@
...
+
+```sh
+astro workspace user remove
+```
+
+## Options
+
+| Option    | Description                                | Possible Values |
+| --------- | ------------------------------------------ | --------------- |
+| `<email>` | The email for the user you want to remove. | Any valid email |
```
Comment (paolaperaza) [inline astro/cli/astro-workspace-user-remove.md:38]: Has the required flag below -- add?

Comment (paolaperaza) [reply astro/cli/astro-workspace-user-remove.md:38]: Should `--workspace-id` be here?

File: astro/cli/astro-workspace-user-update.md [hunk 14]
```diff
@@ -0,0 +1,70 @@
...
+
+<Tabs
+    defaultValue="astro"
+    values={[
+        {label: 'Astro', value: 'astro'},
+        {label: 'Software', value: 'software'},
+    ]}>
+<TabItem value="astro">
+
+Update a user's permissions in your current Astrp Workspace. The CLI prompts you for the user's email.
```
Comment (paolaperaza) [inline astro/cli/astro-workspace-user-update.md:26]:
```suggestion
Update the role of an existing user in your current Workspace. The CLI prompts you for the user's email.
```

Does it prompt you for other commands? Should we specify that in the other docs?

File: astro/cli/astro-workspace-user-update.md [hunk 15]
```diff
@@ -0,0 +1,70 @@
...
+
+```sh
+astro workspace user update --role <user-role>
+```
+
+## Options
+
+| Option                | Description                                          | Valid Values                                                                     |
+| --------------------- | ---------------------------------------------------- | -------------------------------------------------------------------------------- |
+| `<email>`             | The email for the user you want to invite or update. | Any valid email                                                                  |
```
Comment (paolaperaza) [inline astro/cli/astro-workspace-user-update.md:38]:
```suggestion
| `<email>`             | The email address of the user whose role you want to update. | Any valid email                                                                  |
```

File: astro/cli/astro-workspace.md [hunk 16]
```diff
@@ -1,413 +0,0 @@
----
-sidebar_label: "astro workspace"
```
Comment (paolaperaza) [inline astro/cli/astro-workspace.md:2]: Do we need redirect for this file?

File: sidebarsAstro.js [hunk 17]
```diff
@@ -271,7 +271,21 @@ module.exports = {
...
+          label: "astro workspace",
+          items: [
+            "cli/astro-workspace-list",
+            "cli/astro-workspace-service-account",
+            "cli/astro-workspace-switch",
+            "cli/astro-workspace-team",
+            "cli/astro-workspace-user-add",
+            "cli/astro-workspace-user-list",
+            "cli/astro-workspace-user-remove",
+            "cli/astro-workspace-user-update",
```
Comment (paolaperaza) [inline sidebarsAstro.js:285]: These are alphabetical order? Seems yes, nice

File: static/_redirects [hunk 18]
```diff
@@ -169,6 +169,8 @@
 /astro/cli/get-started /astro/cli/install-cli 301
 /astro/cli/astro-organization-list /astro/cli/astro-organization 301
 /astro/cli/astro-organization-switch /astro/cli/astro-organization 301
+/astro/cli/astro-workspace /astro/cli/reference 301
```
Comment (paolaperaza) [inline static/_redirects:172]: nice

**Review by paolaperaza — APPROVED** [review]
A couple comments here @jwitz but giving you the ✅ ! Will trust you'll look into these

File: astro/cli/astro-workspace-list.md (hunk 2 above)
Comment (jwitz) [reply astro/cli/astro-workspace-list.md:13]: I could see using it for other purposes too, I don't think we need to be explicit here if we have the related command right underneath the usage.

File: astro/cli/astro-workspace-switch.md [hunk 19]
```diff
@@ -0,0 +1,26 @@
...
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+:::info  
+
+The behavior and format of this command differs depending on what Astronomer product you're using. Use the following tabs to change product contexts. 
+
+:::
```
Comment (jwitz) [inline astro/cli/astro-workspace-switch.md:16]:
```suggestion
```

File: astro/cli/astro-workspace-user-list.md (hunk 9 above)
Comment (jwitz) [inline astro/cli/astro-workspace-user-list.md:26]:
```suggestion
List all users with access to your current Workspace.
```

File: astro/cli/astro-workspace-user-list.md (hunk 11 above)
Comment (jwitz) [reply astro/cli/astro-workspace-user-list.md:56]: None of these are required, the CLI will assume your current Workspace when it generates the list.

File: astro/cli/astro-workspace-user-remove.md (hunk 13 above)
Comment (jwitz) [reply astro/cli/astro-workspace-user-remove.md:38]: That's only a flag in Software

File: astro/cli/astro-workspace-user-remove.md (hunk 12 above)
Comment (jwitz) [reply astro/cli/astro-workspace-user-remove.md:26]: I'll ask Piotr about this, my understanding is that it's pretty straightforward: When you remove a user, our auth system kicks them off the UI/ Astro immediately, and they're still a part of the Org

File: astro/cli/astro-workspace-user-update.md (hunk 14 above)
Comment (jwitz) [reply astro/cli/astro-workspace-user-update.md:26]: So weirdly this command does not prompt you for the role, only for the email. If you select an email and you don't include a role, it throws an error

File: astro/cli/astro-deployment-delete.md [hunk 20]
```diff
@@ -29,7 +29,7 @@ When you run `astro deployment delete`, you are prompted to select from a list o
 
 :::info
 
-To complete this action, [Workspace Admin](user-permissions.md#workspace-roles) permissions are required.
+To complete this action, [Workspace Owner](user-permissions.md#workspace-roles) permissions are required.
```
Comment (jwitz) [inline astro/cli/astro-deployment-delete.md:32]:
```suggestion
To complete this action, [Workspace Admin](user-permissions.md#workspace-roles) permissions are required.
```

File: astro/cli/release-notes.md [hunk 21]
```diff
@@ -367,7 +367,7 @@ If your CI/CD pipelines currently define one or more Deployment IDs, you may rem
 
 ### Bug fixes
 
-- Fixed an issue where only Workspace Admins could create Deployments
+- Fixed an issue where only Workspace Owners could create Deployments
```
Comment (jwitz) [inline astro/cli/release-notes.md:370]:
```suggestion
- Fixed an issue where only Workspace Admins could create Deployments
```

File: astro/configure-deployment-resources.md [hunk 22]
```diff
@@ -86,7 +86,7 @@ Transferring a Deployment can be helpful when your team needs to change user acc
 
 To transfer a Deployment from one Workspace to another, the Workspaces must be in the same Organization. Transferred Deployments cannot be transferred to a different cluster from the one in which they were created.
 
-Only the users who are members of the target Workspace can access the Deployment after it is transferred. To transfer a Deployment, you must be a Workspace Admin or Editor in both the original Workspace and the target Workspace.
+Only the users who are members of the target Workspace can access the Deployment after it is transferred. To transfer a Deployment, you must be a Workspace Owner or Editor in both the original Workspace and the target Workspace.
```
Comment (jwitz) [inline astro/configure-deployment-resources.md:89]:
```suggestion
Only the users who are members of the target Workspace can access the Deployment after it is transferred. To transfer a Deployment, you must be a Workspace Admin or Editor in both the original Workspace and the target Workspace.
```

File: astro/deploy-code.md [hunk 23]
```diff
@@ -150,7 +150,7 @@ astro deploy --dags
 
 ### Disable DAG-only deploys on a Deployment
 
-If you have Workspace Admin permissions, you can turn off DAG-only deploys for a Deployment at any time if your organization doesn't benefit from faster deploys or prefers a deployment method that is exclusively based on building and deploying your Astro project as a Docker image. When you turn off DAG-only deploys, the way Airflow and Astro read your DAGs changes, and all existing DAGs are removed from your Deployments and they might not appear in the Airflow UI.
+If you have Workspace Owner permissions, you can turn off DAG-only deploys for a Deployment at any time if your organization doesn't benefit from faster deploys or prefers a deployment method that is exclusively based on building and deploying your Astro project as a Docker image. When you turn off DAG-only deploys, the way Airflow and Astro read your DAGs changes, and all existing DAGs are removed from your Deployments and they might not appear in the Airflow UI.
```
Comment (jwitz) [inline astro/deploy-code.md:153]:
```suggestion
If you have Workspace Admin permissions, you can turn off DAG-only deploys for a Deployment at any time if your organization doesn't benefit from faster deploys or prefers a deployment method that is exclusively based on building and deploying your Astro project as a Docker image. When you turn off DAG-only deploys, the way Airflow and Astro read your DAGs changes, and all existing DAGs are removed from your Deployments and they might not appear in the Airflow UI.
```

File: astro/install-azure.md [hunk 24]
```diff
@@ -111,7 +111,7 @@ From here, Astronomer support will set up the VNet connection to Astro.
     - To authenticate with your GitHub account, click **Continue with GitHub**, enter your username or email address, enter your password, and then click **Sign in**.
     - To authenticate with your Google account, click **Continue with Google**, choose an account, enter your username and password, and then click **Sign In**.
 
-    If you're the first person in an Organization to authenticate, you're added as a Workspace Admin to a new Workspace named after your Organization. You can add other team members to the Workspace without the assistance of Astronomer support. See [Add a user](add-user.md). To integrate an identity provider (IdP) with Astro, see [Set up an identity provider](configure-idp.md).
+    If you're the first person in an Organization to authenticate, you're added as a Workspace Owner to a new Workspace named after your Organization. You can add other team members to the Workspace without the assistance of Astronomer support. See [Add a user](add-user.md). To integrate an identity provider (IdP) with Astro, see [Set up an identity provider](configure-idp.md).
```
Comment (jwitz) [inline astro/install-azure.md:114]:
```suggestion
    If you're the first person in an Organization to authenticate, you're added as a Workspace Admin to a new Workspace named after your Organization. You can add other team members to the Workspace without the assistance of Astronomer support. See [Add a user](add-user.md). To integrate an identity provider (IdP) with Astro, see [Set up an identity provider](configure-idp.md).
```

File: astro/install-azure.md [hunk 25]
```diff
@@ -265,7 +265,7 @@ Wait for confirmation that the installation is successful before you access Astr
...
```
Comment (jwitz) [inline astro/install-azure.md:268]:
```suggestion
    If you're the first person in an Organization to authenticate, you're added as a Workspace Admin to a new Workspace named after your Organization. You can add other team members to the Workspace without the assistance of Astronomer support. See [Add a user](add-user.md). To integrate an identity provider (IdP) with Astro, see [Set up an identity provider](configure-idp.md).
```

File: astro/install-gcp.md [hunk 26]
```diff
@@ -84,7 +84,7 @@ The CIDR range sizes must be equal to or greater than the default range sizes fo
     - To authenticate with your GitHub account, click **Continue with GitHub**, enter your username or email address, enter your password, and then click **Sign in**.
     - To authenticate with your Google account, click **Continue with Google**, choose an account, enter your username and password, and then click **Sign In**.
 
-    If you're the first person in an Organization to authenticate, you're added as a Workspace Admin to a new Workspace named after your Organization. You can add other team members to the Workspace without the assistance of Astronomer support. See [Add a user](add-user.md). To integrate an identity provider (IdP) with Astro, see [Set up an identity provider](configure-idp.md).
+    If you're the first person in an Organization to authenticate, you're added as a Workspace Owner to a new Workspace named after your Organization. You can add other team members to the Workspace without the assistance of Astronomer support. See [Add a user](add-user.md). To integrate an identity provider (IdP) with Astro, see [Set up an identity provider](configure-idp.md).
```
Comment (jwitz) [inline astro/install-gcp.md:87]:
```suggestion
    If you're the first person in an Organization to authenticate, you're added as a Workspace Admin to a new Workspace named after your Organization. You can add other team members to the Workspace without the assistance of Astronomer support. See [Add a user](add-user.md). To integrate an identity provider (IdP) with Astro, see [Set up an identity provider](configure-idp.md).
```

File: astro/install-gcp.md [hunk 27]
```diff
@@ -195,7 +195,7 @@ Wait for confirmation that the installation is successful before you access Astr
...
```
Comment (jwitz) [inline astro/install-gcp.md:198]:
```suggestion
    If you're the first person in an Organization to authenticate, you're added as a Workspace Admin to a new Workspace named after your Organization. You can add other team members to the Workspace without the assistance of Astronomer support. See [Add a user](add-user.md). To integrate an identity provider (IdP) with Astro, see [Set up an identity provider](configure-idp.md).
```

File: astro/release-notes.md [hunk 28]
```diff
@@ -631,7 +631,7 @@ You can now [create an Astro cluster on GCP](create-cluster.md) in the following
 
 The following user roles have new and modified permissions:
 
-- Organization Owners now have Workspace Admin permissions for all Workspaces in their Organization. This role can now access Organization Workspaces, Deployments, and usage data.
+- Organization Owners now have Workspace Owner permissions for all Workspaces in their Organization. This role can now access Organization Workspaces, Deployments, and usage data.
```
Comment (jwitz) [inline astro/release-notes.md:634]:
```suggestion
- Organization Owners now have Workspace Admin permissions for all Workspaces in their Organization. This role can now access Organization Workspaces, Deployments, and usage data.
```

File: astro/release-notes.md [hunk 29]
```diff
@@ -1018,7 +1018,7 @@ The following Organization-level roles are now supported on Astro:
...
 - **Organization Billing Admin:** This role has all of the Organization Member's permissions, plus the ability to manage Organization-level settings and billing. Organization Billing Admins can access the **Usage** tab of the Cloud UI and view all Workspaces across the Organization.
-- **Organization Owner:** This role has all of the Organization Billing Admin's permissions, plus the ability to manage and modify anything within the entire Organization. This includes Deployments, Workspaces, Clusters, and users. Organization Owners have Workspace Admin permissions to all Workspaces within the Organization.
+- **Organization Owner:** This role has all of the Organization Billing Admin's permissions, plus the ability to manage and modify anything within the entire Organization. This includes Deployments, Workspaces, Clusters, and users. Organization Owners have Workspace Owner permissions to all Workspaces within the Organization.
```
Comment (jwitz) [inline astro/release-notes.md:1021]:
```suggestion
- **Organization Owner:** This role has all of the Organization Billing Admin's permissions, plus the ability to manage and modify anything within the entire Organization. This includes Deployments, Workspaces, Clusters, and users. Organization Owners have Workspace Admin permissions to all Workspaces within the Organization.
```

File: astro/release-notes.md [hunk 30]
```diff
@@ -1028,7 +1028,7 @@ All users can now create a new Workspace directly from the **Overview** tab of t
 
 ![Create Workspace button](/img/release-notes/add-workspace.png)
 
-When you create a new Workspace, you will automatically become a Workspace Admin within it and can create Deployments. For more information about managing Workspaces, see [Manage Workspaces](manage-workspaces.md).
+When you create a new Workspace, you will automatically become a Workspace Owner within it and can create Deployments. For more information about managing Workspaces, see [Manage Workspaces](manage-workspaces.md).
```
Comment (jwitz) [inline astro/release-notes.md:1031]:
```suggestion
When you create a new Workspace, you will automatically become a Workspace Admin within it and can create Deployments. For more information about managing Workspaces, see [Manage Workspaces](manage-workspaces.md).
```

File: astro/release-notes.md [hunk 31]
```diff
@@ -1305,7 +1305,7 @@ All user configurations can be found by clicking your user profile picture in th
 ### Additional improvements
 
 - In the Cloud UI, the **Open Airflow** button is now disabled until the Airflow UI of the Deployment is available.
-- Workspace Admins can now edit user permissions and remove users within a given Workspace.
+- Workspace Owners can now edit user permissions and remove users within a given Workspace.
```
Comment (jwitz) [inline astro/release-notes.md:1308]:
```suggestion
- Workspace Admin can now edit user permissions and remove users within a given Workspace.
```

File: astro/user-permissions.md [hunk 32]
```diff
@@ -12,9 +12,9 @@ Astro has hierarchical role based access control. Within a given Workspace or Or
 The Astro role hierarchies in order of inheritance are: 
 
 - Organization Owner > Organization Billing Admin > Organization Member 
-- Workspace Admin > Workspace Editor > Workspace Viewer
+- Workspace Owner > Workspace Editor > Workspace Viewer
 
-Users with Organization Owner permissions also inherit Workspace Admin permissions on all Workspaces.
+Users with Organization Owner permissions also inherit Workspace Owner permissions on all Workspaces.
```
Comment (jwitz) [inline astro/user-permissions.md:17]:
```suggestion
Users with Organization Owner permissions also inherit Workspace Admin permissions on all Workspaces.
```

File: astro/user-permissions.md [hunk 33]
```diff
@@ -32,7 +32,7 @@ An Organization role grants a user some level of access to an Astro Organization
 | Create a new Workspace                                           | ✔️                       | ✔️                              | ✔️                      |
 | Update Organization billing information and settings             |                         | ✔️                              | ✔️                      |
 | View usage for all Workspaces in the **Usage** tab               |                         | ✔️                              | ✔️                      |
-| Workspace Admin permissions to all Workspaces                    |                         |                                | ✔️                      |
+| Workspace Owner permissions to all Workspaces                    |                         |                                | ✔️                      |
```
Comment (jwitz) [inline astro/user-permissions.md:35]:
```suggestion
| Workspace Admin permissions to all Workspaces                    |                         |                                | ✔️                      |
```

File: astro/user-permissions.md [hunk 34]
```diff
@@ -44,7 +44,7 @@ To update user Organization roles, see [Manage users](add-user.md).
 
 A Workspace role grants a user some level of access to a specific Workspace. The following table lists the available Workspace roles:
 
-| Permission                                          | **Workspace Viewer** | **Workspace Editor** | **Workspace Admin** |
+| Permission                                          | **Workspace Viewer** | **Workspace Editor** | **Workspace Owner** |
```
Comment (jwitz) [inline astro/user-permissions.md:47]:
```suggestion
| Permission                                          | **Workspace Viewer** | **Workspace Editor** | **Workspace Admin** |
```

File: learn/cloud-ide-tutorial.md [hunk 35]
```diff
@@ -37,7 +37,7 @@ To get the most out of this tutorial, make sure you have an understanding of:
 ## Prerequisites
 
 - An Astro account. If you are not an Astronomer customer yet and want to learn more about Astro, you can [join the weekly demo](https://www.astronomer.io/events/weekly-demo/) or [contact us directly](https://www.astronomer.io/try-astro/?referral=docs-content-link).
-- A Workspace in which you are either a Workspace Editor or a Workspace Admin.
+- A Workspace in which you are either a Workspace Editor or a Workspace Owner.
```
Comment (jwitz) [inline learn/cloud-ide-tutorial.md:40]:
```suggestion
- A Workspace in which you are either a Workspace Editor or a Workspace Admin.
```

File: astro/cli/astro-workspace-user-add.md [hunk 36]
```diff
@@ -0,0 +1,75 @@
...
+    values={[
+        {label: 'Astro', value: 'astro'},
+        {label: 'Software', value: 'software'},
+    ]}>
+<TabItem value="astro">
+
+Add an existing Organization user to your current Astronomer Workspace. You must be a Workspace Owner to perform this action.
+
+You can use this command to invite multiple users to a Workspace at a time. See [Add a group of users to Astro using the Astro CLI](add-user.md#add-a-group-of-users-to-astro-using-the-astro-cli)
+
```
Comment (jwitz) [inline astro/cli/astro-workspace-user-add.md:29]:
```suggestion
Add an existing Organization user to your current Astronomer Workspace. You must be a Workspace Admin to perform this action.

```

File: astro/add-user.md [hunk 37]
```diff
@@ -5,12 +5,12 @@ id: add-user
...
 ---
 
-As a Workspace Admin or Organization Owner, you can add new team members to Astro and grant them user roles with permissions for specific actions across your Organization. Workspace Admins can remove users from a Workspace, and Organization Owners can remove users from an Organization.
+As a Workspace Admin or Organization Owner, you can add new team members to Astro and grant them user roles with permissions for specific actions across your Organization. Workspace Admin can remove users from a Workspace, and Organization Owners can remove users from an Organization.
 
 ## Prerequisites
 
 - To add, edit, or remove Organization users, you need Organization Owner permissions.
-- To add edit, or remove Workspace users, you need Workspace Admin permissions for a given Workspace. The user must also already be a part of the Organization that hosts the Workspace.
+- To add edit, or remove Workspace users, you need Workspace Owner permissions for a given Workspace. The user must also already be a part of the Organization that hosts the Workspace.
```
Comment (jwitz) [inline astro/add-user.md:13]:
```suggestion
- To add edit, or remove Workspace users, you need Workspace Admin permissions for a given Workspace. The user must also already be a part of the Organization that hosts the Workspace.
```

File: astro/add-user.md [hunk 38]
```diff
@@ -5,12 +5,12 @@ id: add-user
 description: Add, edit, or remove users in the Cloud UI.
 ---
 
-As a Workspace Admin or Organization Owner, you can add new team members to Astro and grant them user roles with permissions for specific actions across your Organization. Workspace Admins can remove users from a Workspace, and Organization Owners can remove users from an Organization.
+As a Workspace Admin or Organization Owner, you can add new team members to Astro and grant them user roles with permissions for specific actions across your Organization. Workspace Admin can remove users from a Workspace, and Organization Owners can remove users from an Organization.
```
Comment (jwitz) [inline astro/add-user.md:8]:
```suggestion
As a Workspace Admin or Organization Owner, you can add new team members to Astro and grant them user roles with permissions for specific actions across your Organization. Workspace Admins can remove users from a Workspace, and Organization Owners can remove users from an Organization.
```

File: astro/release-notes.md [hunk 39]
```diff
@@ -1305,7 +1305,7 @@ All user configurations can be found by clicking your user profile picture in th
 ### Additional improvements
 
 - In the Cloud UI, the **Open Airflow** button is now disabled until the Airflow UI of the Deployment is available.
-- Workspace Admins can now edit user permissions and remove users within a given Workspace.
+- Workspace Admin can now edit user permissions and remove users within a given Workspace.
```
Comment (jwitz) [inline astro/release-notes.md:1308]:
```suggestion
- Workspace Admins can now edit user permissions and remove users within a given Workspace.
```

**Pull request merged**